### PR TITLE
Alpine: Use common-utils Feature instead of a lib script

### PR DESCRIPTION
**Pull request by samruddhikhandale**
Ref: https://github.com/devcontainers/features/pull/326

### CODE REVIEW

File: src/base-alpine/.devcontainer/devcontainer.json [hunk 1]
```diff
@@ -3,15 +3,23 @@
...
 	// "forwardPorts": [],
 
 	// Use 'postCreateCommand' to run commands after the container is created.
 	// "postCreateCommand": "uname -a",
 
 	// Replace when using a ptrace-based debugger like C++, Go, and Rust
-	// "runArgs": [ "--init", "--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined" ],
-
-	// Set `remoteUser` to `root` to connect as root instead. More info: https://aka.ms/vscode-remote/containers/non-root.
-	"remoteUser": "vscode"
```
Comment (Chuxel) [inline src/base-alpine/.devcontainer/devcontainer.json:16]: I assume you still need `"remoteUser": "vscode"`, correct?

Comment (samruddhikhandale) [reply src/base-alpine/.devcontainer/devcontainer.json]: Oops, my bad

**Review by Chuxel — DISMISSED** [review]

File: src/base-alpine/.devcontainer/devcontainer.json [hunk 2]
```diff
@@ -3,6 +3,17 @@
 		"dockerfile": "./Dockerfile",
 		"context": "."
 	},
+
+	"features": {
+		"ghcr.io/devcontainers/features/common-utils:2": {
+			"installZsh": "true",
+			"username": "vscode",
+			"uid": "1000",
+			"gid": "1000",
```
Comment (eitsupi) [inline src/base-alpine/.devcontainer/devcontainer.json:12]:
```suggestion
			"userUid": "1000",
			"userGid": "1000",
```

Comment (Chuxel) [reply src/base-alpine/.devcontainer/devcontainer.json:12]: Good catch! 1000 is the default so it would work, but you're right we should fix that.

Comment (samruddhikhandale) [reply src/base-alpine/.devcontainer/devcontainer.json]: Ah, I missed that. Thanks!

**Review by edgonmsft — APPROVED** [review]

File: src/base-alpine/.devcontainer/devcontainer.json [hunk 3]
```diff
@@ -3,6 +3,17 @@
 		"dockerfile": "./Dockerfile",
 		"context": "."
 	},
+
+	"features": {
```
Comment (samruddhikhandale) [inline src/base-alpine/.devcontainer/devcontainer.json:7]: Waiting for https://github.com/devcontainers/features/pull/388

**Pull request merged**